modules/apps/foundation/frontend-js/frontend-js-aui-web/src/main/resources/META-INF/resources/liferay/item_selector_repository_entry_browser.js

```diff
@@ -188,26 +188,35 @@ AUI.add(
 					_getUploadErrorMessage: function(error) {
 						var instance = this;
 
-						var errorType = error.errorType;
-
 						var message = Liferay.Language.get('an-unexpected-error-occurred-while-uploading-your-file');
 
-						if (errorType === STATUS_CODE.SC_FILE_ANTIVIRUS_EXCEPTION) {
-							message = error.message;
-						}
-						else if (errorType === STATUS_CODE.SC_FILE_EXTENSION_EXCEPTION) {
-							message = Lang.sub(Liferay.Language.get('please-enter-a-file-with-a-valid-extension-x'), [error.message]);
-						}
-						else if (errorType === STATUS_CODE.SC_FILE_NAME_EXCEPTION) {
-							message = Liferay.Language.get('please-enter-a-file-with-a-valid-file-name');
-						}
-						else if (errorType === STATUS_CODE.SC_FILE_SIZE_EXCEPTION || errorType === STATUS_CODE.SC_UPLOAD_REQUEST_CONTENT_LENGTH_EXCEPTION) {
-							message = Lang.sub(Liferay.Language.get('please-enter-a-file-with-a-valid-file-size-no-larger-than-x'), [instance.formatStorage(instance.get('maxFileSize'))]);
-						}
-						else if (errorType === STATUS_CODE.SC_UPLOAD_REQUEST_SIZE_EXCEPTION) {
-							var maxUploadRequestSize = Liferay.PropsValues.UPLOAD_SERVLET_REQUEST_IMPL_MAX_SIZE;
+						if (error && error.errorType) {
```

Comment:
Considering the previous comment (I don't think that we need to consider a use case where no extension is allowed) we shouldn't need this logic either.